app/src/main/java/org/fossasia/openevent/general/auth/SignUpFragment.kt

```diff
@@ -62,6 +64,20 @@ class SignUpFragment : Fragment() {
             redirectToMain()
         })
 
+        rootView.passwordSignUp.addTextChangedListener(object : TextWatcher{
+            override fun afterTextChanged(p0: Editable?) {
+                when {
+                    (passwordSignUp.text.toString().length>=6 || passwordSignUp.text.toString().isEmpty()) -> {
+                        textInputLayoutPassword.error = null
+                        textInputLayoutPassword.isErrorEnabled = false
+                    }
+                    else -> textInputLayoutPassword.error = "Password too short!"
```

Comment:
This should be if else

Reply:
Previously I wrote 'If else' statement here but then kotlin gave me this suggestion(to use when)! If you want I can roll back to 'If else' statement

Reply:
Yes, please move back to if else. When is like switch case and should be used when there are multiple cases